settings/OpeningPeriodForm/BigCalendarWrapper.js

```diff
@@ -136,6 +169,7 @@ class BigCalendarWrapper extends React.PureComponent {
         ...(isAllDay && { title: 'All day' }),
       });
 
+      this.checkEventExistOrOverlap(start, end);
```

Comment:
I wonder why another check is needed here if it already exists [here](https://github.com/folio-org/ui-calendar/pull/232/files#diff-5dad3b4739816352b6df03c31f641704R156).